src/screens/SendConfirmScreen.js

```diff
@@ -52,9 +53,9 @@ class SendConfirmScreen extends React.Component {
   constructor(props) {
     super(props);
     // we receive these 3 values from previous screens
-    this.amount = this.props.navigation.getParam('amount');
-    this.address = this.props.navigation.getParam('address');
-    this.token = this.props.navigation.getParam('token');
+    this.amount = this.props.route.params.amount;
+    this.address = this.props.route.params.address;
+    this.token = this.props.route.params.token;
```

Comment:
These parameters are mandatory, as it wouldn't make sense to have a send confirmation without amount, address and token.
But I agree it would be more helpful to have a weird screen than a crash. Added some protections for missing `params`, but without the `lodash` lib.